app/src/main/java/deakin/gopher/guardian/view/general/Homepage4nurse.kt

```diff
@@ -0,0 +1,39 @@
+package deakin.gopher.guardian.view.general
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import deakin.gopher.guardian.R
+import deakin.gopher.guardian.services.NavigationService
+
+class Homepage4nurse : AppCompatActivity() {
+    @SuppressLint("MissingInflatedId")
```

Comment:
I'd say remove this and see if it still builds and runs. It might not be necessary